ansible/roles/create-vm/templates/ibm-cloud-inventory.j2

```diff
@@ -0,0 +1,11 @@
+{% for item in groups %}
+
+[{{item}}]
+{%   for entry in groups[item] %}
```

Comment:
Could we do something like this? I think jinja templates are basically python and this might be a little more readable IMO.
```suggestion
{% for item, entries in groups.items() %}

[{{item}}]
{%   for entry in entries %}
```

Reply:
that is a good point. I also want to try to make this a bit more readable by having the host variables on individual lines. Might mean converting this to yaml, but I think that's probably fine